app/api/auth/Customer/Register/route.ts

```
@@ -0,0 +1,63 @@
+import { connectToDatabase } from "@/lib/db";
+import Customer from "@/models/Customer";
+import { NextRequest, NextResponse } from "next/server";
+
+export async function POST(req: NextRequest) {
+  console.log("sakdina ma ")
+  try {
+    const {
+      fullName: fullName,
```

Biome (1.9.4)

[error] Useless rename.

Safe fix: Remove the renaming.


(lint/complexity/noUselessRename)